api/topic.go

```diff
@@ -18,17 +18,17 @@ func (api *API) GetTopicHandler(w http.ResponseWriter, req *http.Request) {
 	logdata := log.Data{
 		handlers.CollectionID.Header(): hColID,
 		"request-id":                   ctx.Value(dprequest.RequestIdKey),
-		"image-id":                     id,
+		"topic-id":                     id,
```

Comment:
I the log convention that we use is that logData keys should have underscore (request_id, topic_id), but apparently image-api also uses `-`! Maybe you can make this change in your next PR?

Reply:
Will do. Thank you David.